### ISSUE TITLE
Suggestion: identity health follows actual health percentage

### ISSUE THREAD

**Issue by Venekiel** [Draylar/identity#513]
It would be nice if players could have a consistent health percentage and if scaling only affected max health.

For example, if I go from human to bat with 70% health, I would like to have 70% of a bat's 6 health (3 hearts), and vice versa.

In fact, when I go from human to bat with 70% health, I go to 6 health (3 hearts) anyway.

**Comment by Natedogwoooof** [comment]
Config change the max HP if that's what you mean.

**Comment by Shadowysn** [comment]
What I think Venekiel wants is dynamic scaling of max health between morphs based on percentage similar to how **IChun's Morph Mod** does it.

**Comment by Venekiel** [comment]
No, what i mean is player health should not go to a constant value when they transform, but rather follow the actual health percentage the player has

Like if he has 20 max health, has currently 15 health (75% health), and switches to a bat, he will actually go to 6 health anyway

What i want is instead, make the health of the player go to 6 health x 75%
